stories/molecules/Widget/Widget.stories.jsx

```diff
@@ -34,6 +34,7 @@ export default {
         comparisonText: args({ control: 'text', ...content }),
         withShadow: args({ control: 'boolean', ...appearance }),
         withBorder: args({ control: 'boolean', ...appearance }),
+        withHeaderActions: args({ control: 'boolean', ...appearance }),
```

Comment:
By adding the `withHeaderActions` prop, we adding additional functionality so please remove this prop from appearance to functionality group.